odml/tools/rdf_converter.py

```diff
@@ -56,6 +56,21 @@ def __init__(self, odml_documents):
                     print(err)
                     return
 
+    _conversion_formats = {
```

Comment:
Since this dictionary is an identical subset of the dictionary used in `FormatConverter` maybe it would be a good thing to move both to their own file in the `tools` folder to have them defined only once and import them into `FormatConverter` and `RDFConverter` as required.